BUILD_STEPS.md

```diff
@@ -1,7 +1,7 @@
 
 # Development Instructions for Poller
 
-> This file is used as a reference steps while working on development aspects of SNMP Poller component of Splunk Connect for SNMP!
+> This file is being used as a reference steps while working on development aspects of SNMP Poller component of Splunk Connect for SNMP!
```

Comment:
https://ell.stackexchange.com/questions/23800/is-still-being-used-versus-is-still-used
"It is being used" means that someone is using it at the moment.
I don't see a need to change this.